server/dataGeneration/generateDump.js

```diff
@@ -2,92 +2,61 @@ const fs = require('fs');
 const path = require('path');
 const faker = require('faker');
 const command = require('./command');
+const uuid = require('uuid').v4;
 
-const iterations = 19;
+const iterations = 39;
 
 for (let k = 0; k <= iterations; k++) {
-    let query = '';
+    let authorsQuery = '';
+    let reviewsQuery = '';
 
-    let spacesQuery = '';
-    for (let i = 1 + k * 500000; i <= 500000 + k * 500000; i++) {
-        if (i === 500000 + iterations * 500000) {
-            spacesQuery += `
-workspace-${i}
-\\.
-`;
-        } else {
-            spacesQuery += `
-workspace-${i}`;
+    for (let i = 1 + k * 250000; i <= 250000 + k * 250000; i++) {
+        let first = faker.name.firstName();
+        let last = faker.name.lastName();
+        const email = faker.internet.email();
+        const hash = faker.internet.password();
+
+        while (first.indexOf("'") === 1) {
+            first = faker.name.firstName();
         }
-    }
 
-    let calculationsQuery = '';
+        while (last.indexOf("'") === 1) {
+            last = faker.name.lastName();
+        }
 
-    let reviewsQuery = '';
+        const authorId = uuid();
 
-    for (let i = 1 + k * 500000; i <= 500000 + k * 500000; i++) {
-        const calculation = {
-            id: i,
-            reviewCount: 0,
-            avg: 0,
-            total: 0,
-            spaceId: i,
-        };
+        if (i === 250000 + iterations * 250000) {
+            authorsQuery += `
+${authorId}	${first}	${last}	${email}	${hash}
```

Comment:
Interesting indentation here for the queries. Is this a Postgres convention?

Reply:
Yes, the file has to have the correct spacing, so my solution here was to force it to the beginning. I think their is a more elegant way of writing me script, I may revisit and see if I can clean it up.